src/Framework/Framework/Controls/KnockoutHelper.cs

```diff
@@ -131,16 +128,11 @@ string getHandlerScript()
                 var validationPath = expression is IStaticCommandBinding ? null : GetValidationTargetExpression(control);
                 return GetPostBackHandlersScript(control, propertyName,
                     // validation handler
-                    validationPath == null ? null :
-                    validationPath == RootValidationTargetExpression ? "\"validate-root\"" :
-                    validationPath == "$data" ? "\"validate-this\"" :
-                    $"[\"validate\", {{path:{JsonConvert.ToString(validationPath)}}}]",
+                    validationPath == null ? null : $"[\"validate\", {{path:{JsonConvert.ToString(validationPath)}}}]",
```

Comment:
I'd vote for keeping the validate-root shortcut - it reduces the number of quotes in the expression

Reply:
`validate-root` is returned